recipes/bluesky-darkframes/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bluesky-darkframes" %}
+{% set version = "0.4.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_value = "0e8d2e4eefe74ea439630f53b36a4e6850c5dfa43a16e87f28745b5b57ece4b1" %}
```

Comment:
Please use https://pypi.org/project/grayskull/ to generate the recipe. We would like to avoid the excess of unnecessary jinja  variables here and grayskull will do the right thing for you..